packages/sql/test/__snapshots__/fixtures.spec.ts.snap

```diff
@@ -8,6 +8,189 @@ WITH
 "
 `;
 
+exports[`parser and printer > should format all fixtures > 233.sql 1`] = `
+"CREATE TABLE
+  \\"test\\" (\\"id\\" uuid NOT NULL)
+WITH
+  (oids = false);
+"
+`;
+
+exports[`parser and printer > should format all fixtures > 277.sql 1`] = `
```

Comment:
@nene Is this expected? I'm following https://github.com/sql-formatter-org/sql-formatter#my-sql-contains-templating-syntax-which-sql-formatter-fails-to-parse at https://github.com/un-ts/prettier/pull/301/files#diff-412bd64af2a1add9795f0f4806f1db87c422f799e7dc4bb2a83babafe0f65ab8R19-R24, but there is still an error on formatting. (We're still using `sql-formatter@^12.2.4` for now.

Reply:
Yeah, the parameters currently cannot occur inside the `schema.table.column` syntax. As, well, it's not really a place where a prepared-statement parameter could occur. There are many other places where these parameters can't occur. Like inside strings. Or if you'd like to parameterize some SQL keyword - then really all bets are off.

But this specific limitation is easy to lift. Made a quick fix to make it work: https://github.com/sql-formatter-org/sql-formatter/commit/0875fe57867fcea1a6b421ccd0d5022d2ee9cab7

Reply:
Thanks for you confirmation. I'll upgrade `sql-fomatter` in a seperate PR.

Reply:
@nene Hi, when will your last fix commit been released?

Reply:
This is now released as 13.1.0